elastic/kibana
# [AO] Add alertDetailsUrl to APM rule types

Resolves #158268

## Summary

In this PR, I've added `alertDetailsUrl` as an action variable to the APM rule types. Also, the related functional tests were updated to ensure the URL was generated correctly.

|Action|Result|
|---|---|
|![image](https://github.com/elastic/kibana/assets/12370520/680c575a-8bbc-4ead-9a74-d92d25780fcf)|![image](https://github.com/elastic/kibana/assets/12370520/c3b4fd1e-8ffd-43ab-8d58-130d70cc7767)|

#### Filtered alert
<img src="https://github.com/elastic/kibana/assets/12370520/edd01a7c-2f8e-4bba-92f9-46471d8acb2a" width="800"/>

Note that, for the APM Latency threshold, we already had this variable pointing to the newly implemented alert details page.

## 🧪 How to test
-  Create an APM rule (besides the Latency threshold rule)
- Create an action and add `context.alertDetailsUrl` variable in the message
- After an alert is created, check the action message. The link should land on the `Alerts` page filtered for that specific alert instance
